flowzone.yml

```
@@ -2481,9 +2473,8 @@ jobs:
     steps:
       - *getGitHubAppToken
       - *checkoutVersionedSha
-      - *deployToBalenaAction
       - *getReleaseNotes
```

```suggestion
      - << *getReleaseNotes
        env:
          look_back: 2
```
example of providing a custom look_back value from the calling job